workflows/rnavar.nf

```diff
@@ -0,0 +1,431 @@
+include { MULTIQC                       } from '../modules/nf-core/modules/multiqc/main'
+include { CAT_FASTQ                     } from '../modules/nf-core/modules/cat/fastq/main'
+include { GATK4_BASERECALIBRATOR        } from '../modules/nf-core/modules/gatk4/baserecalibrator/main'
+include { GATK4_BEDTOINTERVALLIST       } from '../modules/nf-core/modules/gatk4/bedtointervallist/main'
+include { GATK4_INTERVALLISTTOOLS       } from '../modules/nf-core/modules/gatk4/intervallisttools/main'
+include { GATK4_HAPLOTYPECALLER         } from '../modules/nf-core/modules/gatk4/haplotypecaller/main'
+include { GATK4_MERGEVCFS               } from '../modules/nf-core/modules/gatk4/mergevcfs/main'
+include { GATK4_INDEXFEATUREFILE        } from '../modules/nf-core/modules/gatk4/indexfeaturefile/main'
+include { GATK4_VARIANTFILTRATION       } from '../modules/nf-core/modules/gatk4/variantfiltration/main'
+include { SAMTOOLS_INDEX                } from '../modules/nf-core/modules/samtools/index/main'
+include { TABIX_TABIX as TABIX          } from '../modules/nf-core/modules/tabix/tabix/main'
+include { CUSTOM_DUMPSOFTWAREVERSIONS   } from '../modules/nf-core/modules/custom/dumpsoftwareversions/main'
+
+/*
+========================================================================================
+    IMPORT NF-CORE SUBWORKFLOWS
+========================================================================================
+*/
+
+include { ALIGN_STAR                    } from '../subworkflows/nf-core/align_star'         // Align reads to genome and sort and index the alignment file
+include { MARKDUPLICATES                } from '../subworkflows/nf-core/markduplicates'     // Mark duplicates in the BAM file
+include { SPLITNCIGAR                   } from '../subworkflows/nf-core/splitncigar'        // Splits reads that contain Ns in their cigar string
+include { RECALIBRATE                   } from '../subworkflows/nf-core/recalibrate'        // Estimate and correct systematic bias
+
+/*
+========================================================================================
+    VARIABLES
+========================================================================================
+*/
+
+// Check STAR alignment parameters
+def prepareToolIndices  = params.aligner
+def seq_platform        = params.seq_platform ? params.seq_platform : []
+def seq_center          = params.seq_center ? params.seq_center : []
+
+// Initialize file channels based on params
+dbsnp                   = params.dbsnp             ? Channel.fromPath(params.dbsnp).collect()               : Channel.empty()
+dbsnp_tbi               = params.dbsnp_tbi         ? Channel.fromPath(params.dbsnp_tbi).collect()           : Channel.empty()
+known_indels            = params.known_indels      ? Channel.fromPath(params.known_indels).collect()        : Channel.empty()
+known_indels_tbi        = params.known_indels_tbi  ? Channel.fromPath(params.known_indels_tbi).collect()    : Channel.empty()
```

Comment:
I notice that some channels have the `ch_` prefix but others do not. I would make it more consistent.

Reply:
cool